GoogleForCreators/web-stories-wp
# Shopping: Support drag & drop from library

## Context

Add support for dragging products from the library to the canvas.



## Summary

Adds draggable product image



## Relevant Technical Choices



## To-do

Handle dragging product that already exists on the canvas



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Visit shopping tab
2. Drag product image from shopping panel to the canvas
3. The image upon drop should insert the product that was selected
4. Products without images or that are already on the page should not be draggable (in other words: if the "Plus" icon button is disabled, so should be dragging)


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #11708
